### ISSUE TITLE
Ranger crash at startup: AttributeError: 'NoneType' object has no attribute 'pointer'

### ISSUE THREAD

**Issue by tarasis** [ranger/ranger#2557]
#### Runtime Environment

- Operating system and version: Arch Linux current version
- Terminal emulator and version: 
- Python version: Python version: 3.10.2 (main, Jan 15 2022, 19:56:27) [GCC 11.1.0]
- Ranger version/commit: ranger version: ranger 1.9.3
- Locale: Locale: en_GB.UTF-8

#### Current Behavior

Opening ranger in bash crashes (using it via ssh)

#### Expected Behavior

It doesn't crash.

#### Possible Solutions

Apparently running Ranger in Fish works. After that opening it in bash works.

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3.10/site-packages/ranger/core/fm.py", line 377, in loop
    ui.redraw()
  File "/usr/lib/python3.10/site-packages/ranger/gui/ui.py", line 334, in redraw
    self.finalize()
  File "/usr/lib/python3.10/site-packages/ranger/gui/ui.py", line 391, in finalize
    DisplayableContainer.finalize(self)
  File "/usr/lib/python3.10/site-packages/ranger/gui/displayable.py", line 268, in finalize
    displayable.finalize()
  File "/usr/lib/python3.10/site-packages/ranger/gui/widgets/view_base.py", line 59, in finalize
    col_y = self.main_column.y + self.main_column.target.pointer \
AttributeError: 'NoneType' object has no attribute 'pointer'
```

**Comment by toonn** [comment]
Sorry for taking so long to respond. Trying to catch up to the backlog of issues. Could you try running the master branch, I think this has been fixed? On Arch the easiest way is to install ranger-master from the AUR.

**Comment by WillNilges** [comment]
Seeing the same issue on Fedora 36, trying to start ranger on an ASCII Text document 🤦🏻 

```
freedom:novnc-patched:% ranger ../novnc-pve/debian/docs                                                             <master ✗>
ranger version: ranger 1.9.3
Python version: 3.10.5 (main, Jun  9 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)]
Locale: en_US.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3.10/site-packages/ranger/core/fm.py", line 377, in loop
    ui.redraw()
  File "/usr/lib/python3.10/site-packages/ranger/gui/ui.py", line 334, in redraw
    self.finalize()
  File "/usr/lib/python3.10/site-packages/ranger/gui/ui.py", line 391, in finalize
    DisplayableContainer.finalize(self)
  File "/usr/lib/python3.10/site-packages/ranger/gui/displayable.py", line 268, in finalize
    displayable.finalize()
  File "/usr/lib/python3.10/site-packages/ranger/gui/widgets/view_base.py", line 59, in finalize
    col_y = self.main_column.y + self.main_column.target.pointer \
AttributeError: 'NoneType' object has no attribute 'pointer'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

```
freedom:novnc-patched:% file ../novnc-pve/debian/docs                                                               <master ✗>
../novnc-pve/debian/docs: ASCII text
```

**Comment by MasterTonLab** [comment]
I have the same issue on my Fedora 35.
OS: Linux fedora35comp 5.15.14-200.fc35.x86_64 #1 SMP Tue Jan 11 16:49:27 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
ranger version: ranger 1.9.3
Python version: 3.10.1 (main, Dec  9 2021, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)]
Locale: en_US.UTF-8

~/.config/ranger/rc.conf
set save_tabs_on_exit true

I open Ranger in my terminal Sakura, it automatically opens a new tab with my home directory (/home/myusername/). And also I see saved tabs.
When trying to go to a saved tab with the Tab key, the Ranger crashes with an error.

And I realized what was the reason for the Ranger crash.
The fact is that the target path of the saved tab is on a drive that is not mounted.
I did the mount:
$ sudo mount -t xfs /dev/sdg4 /mnt/MyNewDisk4
Then I run Ranger and it no longer crashed when switching to all saved tabs.

But this kind of work of the Ranger is not correct. It need to fix this.
